samples/net/cellular_modem/boards/nrf9160dk_nrf9160_ns.conf

```diff
@@ -0,0 +1,16 @@
+CONFIG_UART_ASYNC_API=y
+
+# Align with the Serial LTE Modem (SLM) application.
+CONFIG_MODEM_CELLULAR_CMUX_MAX_FRAME_SIZE=1500
+CONFIG_MODEM_CELLULAR_UART_BUFFER_SIZES=1500
+
+# Allow large UART TXs to go through @115200.
+CONFIG_MODEM_BACKEND_UART_ASYNC_TRANSMIT_TIMEOUT_MS=200
+
+# Prevent sockets getting offloaded to the modem.
+CONFIG_NET_SOCKETS_OFFLOAD=n
```

Comment:
For the demo purposes, wouldn't it be more spectacular to run the modem sample on a host, that does not have built-in modem on it's own (`nrf52840` for instance)? Up for discussion, but for me it seems it could be a bit confusing, having two nRF91 devices connected, to follow how actually the connectivity is provided.

Reply:
Yeah you're right. My problem here is that I only have nRF91 devices with me, so I couldn't test with any other.
One thing we are thinking about is making a configuration so that the nRF52840 that is in the nRF9160 DK can be the host, communicating with the nRF9160 via a UART. Though it's still on the same board.

Reply:
FYI I added a configuration for nRF9160 DK's nRF52840 [here](https://github.com/zephyrproject-rtos/zephyr/pull/69288/commits/5218ac55dc24e8133ecc3d8d2032164964d38c8b).
A next step here would be to add a configuration for an nRF device that doesn't live on a cellular DK.